src/sentry/eventtypes/error.py

```diff
@@ -52,7 +80,10 @@ def compute_title(self, metadata):
             if value:
                 title += f": {truncatechars(value.splitlines()[0], 100)}"
 
-        return compute_title_with_tree_label(title, metadata)
+        if not metadata.get("hide_tree_labels"):
+            return compute_title_with_tree_label(title, metadata)
+
+        return title or metadata.get("function") or "<unknown>"
```

Comment:
in the frontend the logic is ` title: metadata.type || metadata.function || '<unknown>'`, title having priority only if the flag "custom-event-title" is enabled. should we be consistent here?

Reply:
we can't query the featureflag here at all

Reply:
Actually I believe we already can remove the featureflag right now since access to the mobile strategy is gated by EA alreayd

Reply:
> we can't query the featureflag here at all because of performance

in this case, can we change this logic to be `title or metadata.get("type")  or metadata.get("function") or "<unknown>"`?

Reply:
`title` is assigned to `type` at the beginning of this function, and `metadata.title` is handled in a wrapping function `get_title`... this logic is equivant to frontend